cms/static/js/models/xblock_info.js

```diff
@@ -186,10 +190,50 @@ function(Backbone, _, str, ModuleUtils) {
         },
 
         /**
+         * Return true if subsection content is required to show.
+         * currently, check subsection for an entrance exam.
+        */
+        showSubSectionContent: function() {
+           var type = this.getOverrideType();
+
+           if(type !=null) {
+               //hide the subsection if block is an entrance exam.
+               if (_.has(type, 'is_entrance_exam_subsection') && type['is_entrance_exam_subsection']) {
+                   return false;
+               }
+           }
+           return true;
+       },
+
+        /**
+         * Return the type of xblock.
+        */
+       getOverrideType: function() {
+           var type = null;
+           if(this.get('override_type') != null) {
+               var type = this.get('override_type');
+           }
+           return type;
+        },
+
+       getExamScore: function() {
```

Comment:
Similar to my other comments in this file, things that are specific to entrance exams shouldn't be in this file. Instead, general concepts such as "deletable" or "explanatory message" should be used. The desire here is to show a bit of text beneath the Section title similar to the "released date" text. So I would recommend brainstorming how this text could be passed through in a general way, instead of adding a method to this class specifically about an exam score. In fact, perhaps the "released date" text is just an example of this.